src/parser/parser-factory.ts

```diff
@@ -0,0 +1,10 @@
+import XlsxParser from './xlsx/xlsx-parser';
+
+export default class ParserFactory {
+    create(parserType: string | undefined) {
```

Comment:
We should not use undefined here, we can always send parser type to this function
Please suggest @nirmal sir